docs/intro/installation.rst

```diff
@@ -1,22 +1,33 @@
 Installation
 ============
 
-0. Currently we support only `Linux` platform and `Python 3.6` (**Python 3.5 is not supported!**)
+0. Currently we support ``Linux`` and ``Windows`` platforms and ``Python 3.6``
+    * ``Python 3.5`` is not supported!
+    * ``Windows`` platform requires ``Git`` for Windows (for example, `git <https://git-scm.com/download/win>`__ ), ``Visual Studio 2015/2017`` with ``C++`` build tools installed!
 
-1. Create a virtual environment with `Python 3.6`:
+1. Create a virtual environment with ``Python 3.6``:
 
-   .. code:: bash
+::
```

Comment:
add indent for both `::` (or `.. code:: bash` if you decide to return it) and the code itself so that the code would render inside a list item